Common/Data/Custom/NullData.cs

```diff
@@ -0,0 +1,56 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+
+namespace QuantConnect.Data.Custom
+{
+    /// <summary>
+    /// Represents a custom data type that works as a heartbeat of data in live mode
+    /// </summary>
+    public class NullData : BaseData
+    {
+        /// <inheritdoc/>
```

Comment:
nit - I believe we were not using `<inheritdoc/>` on purpose, I _think_ because it broke the documentation (maybe @StefanoRaggi or @Jay-Jay-D remember)